playwright/drawer-panel-dropdown.spec.ts

```
@@ -0,0 +1,334 @@
+    await drawerHelpers.bulkSelectNone(page);
+
+    // Select 1 notification
+    await drawerHelpers.selectNotification(items.first());
+
+    // Verify count is 1
+    const actionsToggle = page.locator('#notifications-actions-toggle');
+    await actionsToggle.click();
+    const dropdown = page.locator('#notifications-actions-dropdown');
+    await expect(dropdown.getByRole('menuitem', { name: /Mark selected .* as read/ })).toContainText(
+      '(1)'
+    );
```

ESLint

[error] Replace `dropdown.getByRole('menuitem',·{·name:·/Mark·selected·.*·as·read/·})).toContainText(⏎······'(1)'⏎····` with `⏎······dropdown.getByRole('menuitem',·{·name:·/Mark·selected·.*·as·read/·})⏎····).toContainText('(1)'`

(prettier/prettier)